docs/reference/commandline/service_create.md

```diff
@@ -434,7 +435,8 @@ The following options can only be used for bind mounts (`type=bind`):
     <td><b>bind-nonrecursive</b></td>
     <td>
       By default, submounts are recursively bind-mounted as well. However, this behavior can be confusing when a
-      bind mount is configured with <tt>readonly</tt> option, because submounts are not mounted as read-only.
+      bind mount is configured with <tt>readonly</tt> option, because submounts are not mounted as read-only
+      if Docker Engine is older than v25, or Linux kernel is older than v5.12.
```

Comment:
We should probably use a full version for these `v25.0.0`, but probably we should consider rewriting this section to assume "latest" (it now works).